### PR TITLE
Cardano (Wallet) removal

### PR DESCRIPTION
**Pull request by lilfade**
Test for removal of the wallet code and associated tables. This should clear it all out, though it is only tested on a local site and needs testing to ensure it works properly.

### CODE REVIEW

**Comment by nobodyatroot** [comment]
A fresh install on 3 federated instances seems to behave correctly, but as discussed in matrix chat... figure out why the migration is touching the messenger_messenges and rememberme_token tables, tagging @e-five256 since it was pointed out by them.

File: migrations/Version20231019023030.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('DROP SEQUENCE cardano_tx_id_seq CASCADE');
+        $this->addSql('DROP SEQUENCE cardano_tx_init_id_seq CASCADE');
+        $this->addSql('CREATE TABLE messenger_messages (id BIGSERIAL NOT NULL, body TEXT NOT NULL, headers TEXT NOT NULL, queue_name VARCHAR(190) NOT NULL, created_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, available_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, delivered_at TIMESTAMP(0) WITHOUT TIME ZONE DEFAULT NULL, PRIMARY KEY(id))');
```
Comment (melroy89) [inline migrations/Version20231019023030.php:25]: You shall leave `messenger_messages` alone ..

File: migrations/Version20231019023030.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        $this->addSql('ALTER TABLE cardano_tx ADD CONSTRAINT fk_f74c620ecd53edb6 FOREIGN KEY (receiver_id) REFERENCES "user" (id) NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('ALTER TABLE cardano_tx ADD CONSTRAINT fk_f74c620ef624b39d FOREIGN KEY (sender_id) REFERENCES "user" (id) NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('ALTER TABLE cardano_tx ADD CONSTRAINT fk_f74c620eba364942 FOREIGN KEY (entry_id) REFERENCES entry (id) NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('ALTER TABLE cardano_tx_init ADD CONSTRAINT fk_973316583eb84a1d FOREIGN KEY (magazine_id) REFERENCES magazine (id) ON DELETE CASCADE NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('ALTER TABLE cardano_tx_init ADD CONSTRAINT fk_97331658a76ed395 FOREIGN KEY (user_id) REFERENCES "user" (id) NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('ALTER TABLE cardano_tx_init ADD CONSTRAINT fk_97331658ba364942 FOREIGN KEY (entry_id) REFERENCES entry (id) NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('DROP TABLE messenger_messages');
```
Comment (melroy89) [inline migrations/Version20231019023030.php:78]: Do NOT drop `messenger_messages`!?

Comment (nobodyatroot) [reply migrations/Version20231019023030.php:78]: and whatever is going on with `rememberme_token`, probably best to leave that one alone too.

**Review by melroy89 — CHANGES_REQUESTED** [review]
See my remarks above. I think this will break consumer jobs.

**Comment by lilfade** [comment]
> See my remarks above. I think this will break consume jobs.

Indeed, though this diff was ran from main as of last night. We need to find where these were changed in prior commits.

found this: https://github.com/MbinOrg/mbin/commit/99da23da1a3c4e44d07acf52fec325323e494d79 

but this is the addition, doesnt seem to exist in my code.or main: https://github.com/MbinOrg/mbin/blob/main/config/packages/security.yaml

Additionally can't find where messages was edited:
```
created_at timestamp(0) without time zone NOT NULL,
available_at timestamp(0) without time zone NOT NULL,
delivered_at timestamp(0) without time zone DEFAULT NULL::timestamp without time zone,
```
and the changes:
```
$this->addSql('ALTER TABLE messenger_messages ALTER created_at TYPE TIMESTAMP(0) WITHOUT TIME ZONE');
$this->addSql('ALTER TABLE messenger_messages ALTER available_at TYPE TIMESTAMP(0) WITHOUT TIME ZONE');
$this->addSql('ALTER TABLE messenger_messages ALTER delivered_at TYPE TIMESTAMP(0) WITHOUT TIME ZONE');
```

**Comment by melroy89** [comment]
> but this is the addition, doesnt seem to exist in my code.or main: https://github.com/MbinOrg/mbin/blob/main/config/packages/security.yaml

Because this `token_provider` config is also not part (anymore?) of the develop branch in Codeberg: https://codeberg.org/Kbin/kbin-core/src/branch/develop/config/packages/security.yaml .. Should it be part of our code base? Or did Ernest also removed it again later..?

**Comment by lilfade** [comment]
> > but this is the addition, doesnt seem to exist in my code.or main: https://github.com/MbinOrg/mbin/blob/main/config/packages/security.yaml
> 
> Because this `token_provider` config is also not part (anymore? or never was?) of the develop branch in Codeberg: https://codeberg.org/Kbin/kbin-core/src/branch/develop/config/packages/security.yaml .. Should it be part of our code base? Or did Ernest also removed it again later..?

Suppose that issue is fine then, though if it's removed shouldn't there be a removal of the checkbox from the login or is this different?

The messenger I haven't found where the change was yet, something had to change in entities for this to push this diff out so will look for that yet.

**Comment by melroy89** [comment]
Checkbox is still working and is using sessions to keep the user logged AFAIK.

**Comment by melroy89** [comment]
Fixed file conflict for you.

**Comment by nobodyatroot** [comment]
Fwiw, I ran the latest main branch in a docker test instance with random data generated from the test fixtures, pointing at the PR branch, and rebuilt while keeping the database intact... everything survived the migration. I don't see any issues moving forward with this PR. @melroy89, it looks like you've requested changes? I believe those changes have been addressed with the recent commits from @lilfade.

**Review by melroy89 — APPROVED** [review]

**Comment by melroy89** [comment]
I also removed "WIP" from the title, just note that GitHub doesn't have WIP PRs. In GitHub there are "Draft Pull Requests" instead. @lilfade see my PM to you in Matrix.

File: migrations/Version20231019023030.php [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20231019023030 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
```
Comment (melroy89) [inline migrations/Version20231019023030.php:17]: Feel free to add some description here. And remove all the generated comments.

**Comment by melroy89** [comment]
Today I will review it again and run in acceptance

**Comment by melroy89** [comment]
@lilfade also note to yourself: I didn't need to fork the project. You are owner of the organization and this repository. You could just have created a new branch.

**Comment by nobodyatroot** [comment]
> @lilfade also note to yourself: I didn't need to fork the project. You are owner of the organization and this repository. You could just have created a new branch.

also makes it easier to test just fyi... easy to change a branch vs. fetching a new origin, etc, etc.

**Comment by melroy89** [comment]
> > @lilfade also note to yourself: I didn't need to fork the project. You are owner of the organization and this repository. You could just have created a new branch.
> 
> also makes it easier to test just fyi... easy to change a branch vs. fetching a new origin, etc, etc.

Exactly, that was maybe the story behind me remark. Thanks for adding this,

**Comment by lilfade** [comment]
Yea will keep that in mind for future commits.

**Comment by melroy89** [comment]
> Today I will review it again and run in acceptance

didn't see anything major happening.

**Review by melroy89 — APPROVED** [review]

**Pull request merged**